apollo-portal/src/main/resources/static/scripts/controller/NamespaceController.js

```diff
@@ -66,7 +66,7 @@ namespace_module.controller("LinkNamespaceController",
                 appId: $scope.appId,
                 name: '',
                 comment: '',
-                isPublic: true,
+                isPublic: false,
```

Comment:
> make create private namespace as default type

Will this change bring confusion to the user as the default behavior is changed?

Reply:
Yes，I think the most scene is creating private namespace not public namespace. 

I changed select namespace type field above input namespace name field. So the user clearly knows the type of namespace being created.